### PR TITLE
Initial support for csv import rake task and correctly rendered in me…

### PR DESCRIPTION
**Pull request by drewpotter**
…tadata index page. Also fixed a typo in the variable within the JavaScript code.

### CODE REVIEW

File: lib/tasks/import_metadata.rake [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      metadata.version = metadata_row[metadata_hash[:version]]&.strip || "Empty"
+      metadata.contact_organisation = metadata_row[metadata_hash[:contact_organisation]]&.strip || "Empty"
+      metadata.website_download_link = metadata_row[metadata_hash[:website_download_link]]&.strip || "Empty"
+      metadata.metadata = "Empty"
+      metadata.factsheet = metadata_row[metadata_hash[:factsheet]]&.strip || "Empty"
+      metadata.marine_spatial_planning = metadata_row[metadata_hash[:marine_spatial_planning]]&.strip || "Empty"
+      metadata.education = metadata_row[metadata_hash[:education]]&.strip || "Empty"
+      metadata.environmental_impact_assessment = metadata_row[metadata_hash[:environmental_impact_assessment]]&.strip || "Empty"
+      metadata.ecosystem_assessment = metadata_row[metadata_hash[:ecosystem_assessment]]&.strip || "Empty"
+      metadata.ecosystem_services = metadata_row[metadata_hash[:ecosystem_services]]&.strip || "Empty"
```
Comment (Levia) [inline lib/tasks/import_metadata.rake]: There should be a way of simplifying all those assignments to just a couple of lines of code.
Usually, when you have to populate an object, you can do something like: 
`Metadata.new(hash)`

That `hash` is a populated set of keys and values for the object, so, for example, it has `{category: value, ...}`. As soon as the keys are the correct column names and the values are in the correct format, the object gets populated.

Comment (drewpotter) [reply lib/tasks/import_metadata.rake]: Thanks @Levia that is a much better way of doing things. I guess I can still clean the data using `strip` and replacing with "Empty" if necessary using that technique?

File: lib/tasks/import_metadata.rake [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      metadata: nil,
+      factsheet: csv_headers[7],
+      marine_spatial_planning: csv_headers[8],
+      education: csv_headers[9],
+      environmental_impact_assessment: csv_headers[10],
+      ecosystem_assessment: csv_headers[11],
+      ecosystem_services: csv_headers[12]
+    }
+
+    CSV.parse(csv, :headers => true, encoding: "utf-8") do |row|
```
Comment (Levia) [inline lib/tasks/import_metadata.rake]: I've just noticed this. Can you please use the latest ruby syntax?
So change `:headers => true` to `headers: true`

Comment (drewpotter) [reply lib/tasks/import_metadata.rake]: @Levia thanks I have made this alteration.

**Pull request merged**